main.go

```diff
@@ -0,0 +1,200 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"net/http/cookiejar"
+	"os"
+	"time"
+
+	"github.com/xuri/excelize/v2"
+)
+
+const (
+	maxPasswordAgeDays int = 30
+)
+
+type Input struct {
+	Filename                     string
+	SheetName                    string
+	UsernameHeader               string
+	PasswordHeader               string
+	AutomatedSheetPassword       string
+	automatedSheetName           string // sheet managed by application
+	automatedSheetColNameToIndex map[string]int
+	rowOffset                    int // number of header rows (common to all sheets)
+	sheetOffset                  int // offset to 0-based coords since sheet coords are 1-based
```

Comment:
is this something that varies by spreadsheet? or is this a property of the library/API we are using to edit the spreadsheets? if it's the latter then I don't think it is part of the input.

Reply:
No, does not vary by spreadsheet. No, is not part of API. I removed `sheetOffset` everywhere. If I need to convert from a zero-based coordinate to a 1-based coordinate, I call a function toSheetCoord(xCoord int) that converts by adding 1.